src/logic/operate.js

```diff
@@ -0,0 +1,12 @@
+import Big from 'big.js';
+
+export default function operate(numOne, numTwo, operation) {
+  const x = Big(numOne);
+  const y = Big(numTwo);
+  if (operation === '+') return x.plus(y);
+  if (operation === '-') return x.minus(y);
+  if (operation === '*') return x.times(y);
+  if (operation === '/') return x.div(y);
```

Comment:
### [Optional]
It is a good practice to use `switch case` once our condition is more than 3;
kindly check out this article 👉  https://medium.com/the-daily-standup/javascript-switch-statement-vs-if-else-if-ee3111793fa0